locale/update_translations.sh

```diff
@@ -57,10 +58,9 @@ done
 
 cd ${srcdir}
 
-echo "Translations compiled successfully. Now some manual steps:"
-echo " 1. less BOINC-Manager-pofilter.txt # check output from pofilter and adjust translations then start this script again"
-echo " 2. git add -u # only update already tracked files (add new files when needed too)"
-echo " 3. git commit -m \"Locale: Update localization files [skip ci]\""
-echo " 4. git push"
+git add .
+git diff-index --cached --quiet HEAD || { git commit -m "Locale: Update localization files"; git push origin HEAD; }
```

Comment:
Now I see the code is change here. How it push without a password?

Reply:
GitHub Actions have embedded credentials, so when you do a push from the CI, you are able to push commits to the repo from which CI is run.